src/components/FullScreenGallery.module.scss

```diff
@@ -1,7 +1,7 @@
 @import "styles/modules/all";
 
 .fullScreen {
-  width: 100vw;
+  width: 100%;
```

Comment:
This fixes the issue on desktop where the right arrow and the close button is getting cut off on the full screen gallery.